selotape/startup.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Do not add extra code/files as place holders.

Reply:
OK